### ISSUE TITLE
Format Error

### ISSUE THREAD

**Issue by akira-toriyama** [mskelton/dtsfmt#13]
Thank you for the great app.
`hold-while-undecided` collapses.

- before
```
#define X_DOWN_ARROW_T_RR LEFT_CONTROL RG(RA(K_APPLICATION))

// ------------------------------------
// https://github.com/zmkfirmware/zmk/pull/1811
&mt {
  hold-while-undecided;
};
/ {
  combos {
```

- after
```
#define X_DOWN_ARROW_T_RR LEFT_CONTROL RG(RA(K_APPLICATION))

// ------------------------------------
// https://github.com/zmkfirmware/zmk/pull/1811
&mt {
  hold-while-undecided = ;
};
/ {
  combos {
```

https://github.com/akira-toriyama/Adv360-Pro-ZMK/blob/feature/r/config/adv360.keymap#L220-L227

**Comment by mskelton** [comment]
Fixed in 37d29553

**Comment by akira-toriyama** [comment]
Thank you.
